cmd/cerbosctl/get/internal/flagset/sort.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021-2022 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package flagset
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+)
+
+const SortByFlag = "sort-by"
+
+type Sort struct {
+	SortBy SortByFlagType
+}
+
+func (s *Sort) FlagSet() *pflag.FlagSet {
+	fs := pflag.NewFlagSet("sort", pflag.ExitOnError)
+	fs.Var(&s.SortBy, SortByFlag, "Sort policies by column")
+	return fs
+}
+
+type SortByValue string
+
+const (
+	SortByPolicyID SortByValue = "policyId"
+	SortByName     SortByValue = "name"
+	SortByVersion  SortByValue = "version"
+)
+
+func (sbv SortByValue) String() string {
+	return string(sbv)
+}
+
+type SortByFlagType SortByValue
+
+func (kf SortByFlagType) Kind() SortByValue {
+	return SortByValue(kf)
+}
+
+func (kf SortByFlagType) String() string {
+	return SortByPolicyID.String()
+}
+
+func (kf *SortByFlagType) Set(v string) error {
+	switch v {
+	case SortByPolicyID.String(), SortByName.String(), SortByVersion.String():
+		*kf = SortByFlagType(v)
+		return nil
```

Comment:
Because `SortByValue` is a type alias of `string`, it would be more efficient to simply cast it than call a method on it.
```suggestion
	switch SortByValue(v) {
	case SortByPolicyID, SortByName, SortByVersion:
		*kf = SortByFlagType(v)
		return nil
```